docs/DeveloperGuide.md

```diff
@@ -1,29 +1,104 @@
-# Developer Guide
+# Developer Guide - Voyagers
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+This project is a team effort by the NUS CS2113 Group W14-4
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+The Voyagers application is designed as a command-line interface 
+(CLI) tool for managing travel trips. 
+It is implemented in Java and follows a modular architecture to promote scalability and maintainability.
 
+The following UML sequence diagram shows the flow of the application when the user enters a command.
+
+![Sequence Diagram](diagrams/seq_diagram_flow.png)
```

Comment:
This implies that the user does the execution of all commands instead of a Main or Parser
<img width="547" alt="usererror" src="https://github.com/nus-cs2113-AY2324S2/tp/assets/111479643/bf29fc56-645d-46a9-9c42-1de18aa4cc3b">